packages/react/select/src/Select.tsx

```diff
@@ -1010,11 +1044,31 @@ const SelectItemText = React.forwardRef<SelectItemTextElement, SelectItemTextPro
     const context = useSelectContext(ITEM_TEXT_NAME, __scopeSelect);
     const contentContext = useSelectContentContext(ITEM_TEXT_NAME, __scopeSelect);
     const itemContext = useSelectItemContext(ITEM_TEXT_NAME, __scopeSelect);
-    const ref = React.useRef<SelectItemTextElement | null>(null);
-    const composedRefs = useComposedRefs(forwardedRef, ref, itemContext.onItemTextChange, (node) =>
-      contentContext.itemTextRefCallback?.(node, itemContext.value, itemContext.disabled)
+    const itemsContext = useSelectNativeOptionsContext(ITEM_TEXT_NAME, __scopeSelect);
```

Comment:
```suggestion
    const nativeOptionsContext = useSelectNativeOptionsContext(ITEM_TEXT_NAME, __scopeSelect);
```

Easier to distinguish perhaps?